tools/wpt/tests/test_wpt.py

```diff
@@ -270,6 +270,26 @@ def test_files_changed(capsys):
     assert err == ""
 
 
+@pytest.mark.xfail(sys.platform == "win32",
+                   reason="Tests currently don't work on Windows for path reasons")
+def test_files_changed_null(capsys):
+    commit = "9047ac1d9f51b1e9faa4f9fad9c47d109609ab09"
+    with pytest.raises(SystemExit) as excinfo:
+        wpt.main(argv=["files-changed", "--null", "%s~..%s" % (commit, commit)])
+    assert excinfo.value.code == 0
+    out, err = capsys.readouterr()
+    assert out == "\0".join(["html/browsers/offline/appcache/workers/appcache-worker.html",
+        "html/browsers/offline/appcache/workers/resources/appcache-dedicated-worker-not-in-cache.js",
+        "html/browsers/offline/appcache/workers/resources/appcache-shared-worker-not-in-cache.js",
+        "html/browsers/offline/appcache/workers/resources/appcache-worker-data.py",
+        "html/browsers/offline/appcache/workers/resources/appcache-worker-import.py",
+        "html/browsers/offline/appcache/workers/resources/appcache-worker.manifest",
+        "html/browsers/offline/appcache/workers/resources/appcache-worker.py",
+        ""
+    ])
```

Comment:
./tests/test_wpt.py:289:5: E124 closing bracket does not match visual indentation

Reply:
Ahah. Got it